source/source-connector/configuration-properties/output-format.txt

```diff
@@ -56,6 +56,14 @@ Settings
        | **Default**: ``json``
        | **Accepted Values**: ``bson``, ``json``, ``schema``
 
+       .. note:: Protobuf Output Format Support
```

Comment:
Suggestion:

Should this be a tip instead of a note? Seems like that might be more appropriate according to the style guide.